From 4bf032f9b18ea6df1ca25e22401c182588b00960 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=A9r=C3=B4me=20Laban?= <jerome.laban@nventive.com>
Date: Sat, 15 Oct 2022 02:27:14 +0000
Subject: [PATCH] fix: adjust for additional EmccExportedFunction

---
 src/mono/wasm/wasm.proj | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/mono/wasm/wasm.proj b/src/mono/wasm/wasm.proj
index 0fa4a43fdc8..c078761909d 100644
--- a/src/mono/wasm/wasm.proj
+++ b/src/mono/wasm/wasm.proj
@@ -128,6 +128,13 @@
       <EmccExportedFunction Include="_htons" />
       <EmccExportedFunction Include="_ntohs" />
     </ItemGroup>
+    <ItemGroup>
+      <EmccExportedRuntimeMethod Include="lengthBytesUTF8" />
+      <EmccExportedRuntimeMethod Include="stringToUTF8" />
+      <EmccExportedRuntimeMethod Include="addFunction" />
+      <EmccExportedRuntimeMethod Include="removeFunction" />
+      <EmccExportedRuntimeMethod Include="IDBFS" />
+    </ItemGroup>
     <!-- after 3.0 -->
     <ItemGroup Condition="$([MSBuild]::VersionGreaterThan('$(_EmccVersion)', '3.0'))">
       <EmccExportedFunction Include="_memset" />
-- 
2.25.1

