From 4d2a655369120cc5de4a67a8b7e4027f44098e8a Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome@platform.uno>
Date: Tue, 9 May 2023 21:07:49 +0000
Subject: [PATCH] disable ccache windows

---
 src/mono/CMakeLists.txt | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/src/mono/CMakeLists.txt b/src/mono/CMakeLists.txt
index 5ff644b7cd5..e7ce2768aad 100644
--- a/src/mono/CMakeLists.txt
+++ b/src/mono/CMakeLists.txt
@@ -42,9 +42,11 @@ set(CMAKE_MODULE_PATH
 set(CMAKE_INSTALL_MESSAGE LAZY)
 
 if(USE_CCACHE)
-  find_program(CCACHE_PROGRAM ccache)
-  if(CCACHE_PROGRAM)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+  if(NOT CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    find_program(CCACHE_PROGRAM ccache)
+    if(CCACHE_PROGRAM)
+      set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+    endif()
   endif()
 endif()
 
-- 
2.25.1

