From dc7db7e8a59dc5aba58bf1f25b70952aa34a9445 Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome@platform.uno>
Date: Mon, 19 Jun 2023 12:50:59 +0000
Subject: [PATCH] fix: Adjust for unused wasm-opt parameter

---
 src/mono/wasm/wasm.proj | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/mono/wasm/wasm.proj b/src/mono/wasm/wasm.proj
index 7bc98cdff51..58f987dad67 100644
--- a/src/mono/wasm/wasm.proj
+++ b/src/mono/wasm/wasm.proj
@@ -307,6 +307,10 @@
       <!-- workaround for https://github.com/emscripten-core/emscripten/issues/18034 -->
       <_EmccLinkFlags Include="-s TEXTDECODER=0"/>
     </ItemGroup>
+ 
+    <ItemGroup Condition="'$(WasmEnableSIMD)' == 'true'">
+      <WasmOptConfigurationFlags Include="--enable-simd" />
+    </ItemGroup>
 
     <ItemGroup Condition="'$(OS)' != 'Windows_NT'">
       <_EmccLinkFlags Include="--profiling-funcs" />
@@ -394,7 +398,6 @@
       <CMakeBuildRuntimeConfigureCmd>$(CMakeBuildRuntimeConfigureCmd) -DICU_LIB_DIR=&quot;$(ICULibDir.TrimEnd('\/').Replace('\','/'))&quot;</CMakeBuildRuntimeConfigureCmd>
       <CMakeBuildRuntimeConfigureCmd>$(CMakeBuildRuntimeConfigureCmd) -DMONO_ARTIFACTS_DIR=&quot;$(MonoArtifactsPath.TrimEnd('\/').Replace('\','/'))&quot;</CMakeBuildRuntimeConfigureCmd>
       <CMakeBuildRuntimeConfigureCmd>$(CMakeBuildRuntimeConfigureCmd) -DNATIVE_BIN_DIR=&quot;$(NativeBinDir.TrimEnd('\/').Replace('\','/'))&quot;</CMakeBuildRuntimeConfigureCmd>
-      <CMakeBuildRuntimeConfigureCmd Condition="'$(WasmEnableSIMD)' == 'true' and '$(Configuration)' == 'Release'">$(CMakeBuildRuntimeConfigureCmd) -DWASM_OPT_ADDITIONAL_FLAGS=&quot;--enable-simd&quot;</CMakeBuildRuntimeConfigureCmd>
       <CMakeBuildRuntimeConfigureCmd Condition="'$(MonoWasmThreads)' == 'true'">$(CMakeBuildRuntimeConfigureCmd) -DDISABLE_THREADS=0</CMakeBuildRuntimeConfigureCmd>
       <CMakeBuildRuntimeConfigureCmd Condition="'$(WasmEnableLegacyJsInterop)' == 'false'">$(CMakeBuildRuntimeConfigureCmd) -DDISABLE_LEGACY_JS_INTEROP=1</CMakeBuildRuntimeConfigureCmd>
       <CMakeBuildRuntimeConfigureCmd>$(CMakeBuildRuntimeConfigureCmd) $(CMakeConfigurationEmsdkPath)</CMakeBuildRuntimeConfigureCmd>
-- 
2.25.1

