From a1264af4f89261f3ada3a71cb3207af40fd99a12 Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome.laban@nventive.com>
Date: Tue, 31 May 2022 13:11:17 +0000
Subject: [PATCH] chore: Update icu transport

---
diff --git a/src/mono/wasm/wasm.proj b/src/mono/wasm/wasm.proj
index 1f08bf6d9c7..dc9b6fc8246 100644
--- a/src/mono/wasm/wasm.proj
+++ b/src/mono/wasm/wasm.proj
@@ -188,7 +188,8 @@
 
     <ItemGroup>
       <ICULibNativeFiles Include="$(ICULibDir)/libicuuc.a;
-                                  $(ICULibDir)/libicui18n.a" />
+                                  $(ICULibDir)/libicui18n.a;
+                                  $(ICULibDir)/libicudata.a" />
       <ICULibFiles Include="$(ICULibDir)/*.dat" />
     </ItemGroup>
     <PropertyGroup>
-- 
2.35.1

