From 335027d83d99e6e27e09199794a56d068cd2f0fe Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome.laban@nventive.com>
Date: Mon, 23 Aug 2021 20:01:39 +0000
Subject: [PATCH] support for IDBFS

---
 src/mono/wasm/wasm.proj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/mono/wasm/wasm.proj b/src/mono/wasm/wasm.proj
index 3eacc3e1307..ce3691ae6a2 100644
--- a/src/mono/wasm/wasm.proj
+++ b/src/mono/wasm/wasm.proj
@@ -66,6 +66,7 @@
       <_EmccCommonFlags Include="-s ALLOW_MEMORY_GROWTH=1" />
       <_EmccCommonFlags Include="-s NO_EXIT_RUNTIME=1" />
       <_EmccCommonFlags Include="-s FORCE_FILESYSTEM=1" />
+      <_EmccCommonFlags Include="-lidbfs.js" />
       <_EmccCommonFlags Include="-s &quot;EXPORTED_RUNTIME_METHODS=['ccall', 'FS_createPath', 'FS_createDataFile', 'cwrap', 'setValue', 'getValue', 'UTF8ToString', 'UTF8ArrayToString', 'addFunction']&quot;" />
       <_EmccCommonFlags Include="-s &quot;EXPORTED_FUNCTIONS=['_putchar']&quot;" />
       <_EmccCommonFlags Include="--source-map-base http://example.com" />
-- 
2.17.1

