From 13f2215d705b87f2aa66c7d9f32f136c20493b27 Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome.laban@nventive.com>
Date: Thu, 19 Nov 2020 17:14:59 +0000
Subject: [PATCH] Enable release assertions, enable LLVM

---
 src/mono/mono.proj | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/mono/mono.proj b/src/mono/mono.proj
index 7726c9b21b2..2279acb2aca 100644
--- a/src/mono/mono.proj
+++ b/src/mono/mono.proj
@@ -183,7 +183,7 @@
     </ItemGroup>
     <!-- WASM specific options -->
     <PropertyGroup Condition="'$(TargetsBrowser)' == 'true'">
-      <_MonoMinimal Condition="'$(Configuration)' == 'Release'">,debugger_agent,assert_messages</_MonoMinimal>
+      <_MonoMinimal Condition="'$(Configuration)' == 'Release'">,debugger_agent</_MonoMinimal>
     </PropertyGroup>
     <ItemGroup Condition="'$(TargetsBrowser)' == 'true'">
       <_MonoCMakeArgs Include="-DENABLE_MINIMAL=jit,portability,sgen_major_marksweep_conc,sgen_split_nursery,sgen_gc_bridge,sgen_toggleref,sgen_debug_helpers,sgen_binary_protocol,logging,shared_perfcounters,interpreter,threads,eventpipe,qcalls$(_MonoMinimal)"/>
-- 
2.17.1
