From e5a3d1639ad48be6e501215c89d7bb9f279ea819 Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome.laban@nventive.com>
Date: Tue, 3 Nov 2020 18:09:54 +0000
Subject: [PATCH 2/2] Enable LLVM Runtime

---
 src/mono/mono.proj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/mono/mono.proj b/src/mono/mono.proj
index fba24bd36c0..c7c4022005d 100644
--- a/src/mono/mono.proj
+++ b/src/mono/mono.proj
@@ -191,6 +191,7 @@
       <_MonoCMakeArgs Include="-DDISABLE_CRASH_REPORTING=1"/>
       <_MonoCMakeArgs Include="-DENABLE_ICALL_EXPORT=1"/>
       <_MonoCMakeArgs Include="-DENABLE_LAZY_GC_THREAD_CREATION=1"/>
+      <_MonoCMakeArgs Include="-DENABLE_LLVM_RUNTIME=1"/>
       <_MonoCFLAGS Include="-I$(PkgMicrosoft_NETCore_Runtime_ICU_Transport)/runtimes/browser-wasm/native/include"/>
     </ItemGroup>
     <!-- iOS/tvOS specific options -->
-- 
2.17.1

