From a25a70289ef7ae82335c6036d392167e408c38b4 Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome.laban@nventive.com>
Date: Sat, 4 Jun 2022 03:36:28 +0000
Subject: [PATCH] fix: Add missing _free export

---
 src/mono/wasm/wasm.proj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mono/wasm/wasm.proj b/src/mono/wasm/wasm.proj
index 605d85774dd..06bcffb3343 100644
--- a/src/mono/wasm/wasm.proj
+++ b/src/mono/wasm/wasm.proj
@@ -131,7 +131,7 @@
 
     <PropertyGroup>
       <_DefaultExportedFunctions Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(_EmccVersion)', '3.1.7'))"
-                                >_malloc,stackSave,stackRestore,stackAlloc,_memalign,_memset,_htons,_ntohs</_DefaultExportedFunctions>
+                                >_malloc,stackSave,stackRestore,stackAlloc,_memalign,_memset,_htons,_ntohs,_free</_DefaultExportedFunctions>
 
       <!-- _htons,_ntohs,__get_daylight,__get_timezone,__get_tzname are exported temporarily, until the issue is fixed in emscripten, https://github.com/dotnet/runtime/issues/64724  -->
       <_DefaultExportedFunctions Condition="'$(_DefaultExportedFunctions)' == '' and $([MSBuild]::VersionGreaterThan('$(_EmccVersion)', '3.0'))"
-- 
2.35.1

