From ffb5d7b59573ae59003264e4039b9d6f811a94cf Mon Sep 17 00:00:00 2001
From: Zoltan Varga <vargaz@gmail.com>
Date: Mon, 26 Oct 2020 12:40:15 -0400
Subject: [PATCH 3/3] Remove unused libicu files.

---
 src/mono/wasm/wasm.proj | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/mono/wasm/wasm.proj b/src/mono/wasm/wasm.proj
index 1848ba8f6d4..b1be12a4fd6 100644
--- a/src/mono/wasm/wasm.proj
+++ b/src/mono/wasm/wasm.proj
@@ -105,10 +105,7 @@
                        $(MonoProjectRoot)wasm\runtime\library_mono.js;
                        $(LibrariesProjectRoot)Native\Unix\System.Native\pal_random.js;
                        $(ArtifactsObjDir)wasm\pinvoke-table.h;
-                       $(_IcuLibdir)\libicudata.a;
                        $(_IcuLibdir)\libicui18n.a;
-                       $(_IcuLibdir)\libicutest.a;
-                       $(_IcuLibdir)\libicutu.a;
                        $(_IcuLibdir)\libicuuc.a"
           DestinationFolder="$(MicrosoftNetCoreAppRuntimePackNativeDir)"
           SkipUnchangedFiles="true" />
-- 
2.17.1

