From 65b67927aa5b31b847ed8c7db2dd3ce5deeb24a5 Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome.laban@nventive.com>
Date: Wed, 1 Jun 2022 15:12:47 +0000
Subject: [PATCH] fix: Adjust for missing atomics

---
 src/mono/wasm/wasm.proj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/mono/wasm/wasm.proj b/src/mono/wasm/wasm.proj
index dc9b6fc8246..605d85774dd 100644
--- a/src/mono/wasm/wasm.proj
+++ b/src/mono/wasm/wasm.proj
@@ -156,6 +156,7 @@
       <_EmccLinkFlags Include="-s MODULARIZE=1"/>
       <_EmccLinkFlags Include="-Wl,--allow-undefined"/>
       <_EmccLinkFlags Include="-s ENVIRONMENT=&quot;web,webview,worker,node,shell&quot;" />
+      <_EmccLinkFlags Condition="'$(MonoWasmThreads)' == 'true'" Include="-matomics" />
     </ItemGroup>
 
     <ItemGroup Condition="'$(OS)' != 'Windows_NT'">
-- 
2.35.1

