From ab2708aabec6645577c917c52a30b7ca21038c7b Mon Sep 17 00:00:00 2001
From: Jerome Laban <jerome@platform.uno>
Date: Tue, 9 May 2023 21:11:35 +0000
Subject: [PATCH] disable bind static warning

---
 src/mono/wasm/runtime/exports.ts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/mono/wasm/runtime/exports.ts b/src/mono/wasm/runtime/exports.ts
index c7ea64155b8..b305e94ed1c 100644
--- a/src/mono/wasm/runtime/exports.ts
+++ b/src/mono/wasm/runtime/exports.ts
@@ -71,7 +71,6 @@ function initializeExports(): RuntimeAPI {
             // eslint-disable-next-line @typescript-eslint/ban-ts-comment
             // @ts-ignore
             module.mono_bind_static_method = (fqn: string, signature: string/*ArgsMarshalString*/): Function => {
-                mono_log_warn("Module.mono_bind_static_method is obsolete, please use [JSExportAttribute] interop instead");
                 return mono_bind_static_method(fqn, signature);
             };
         }
-- 
2.25.1

